docs/xamarin-forms/Xaml-Navigation.md

```diff
@@ -0,0 +1,101 @@
+# Xaml Navigation
+
+Prism allows you to declare your navigation directly inside your Xaml via a Markup Extensions. 
```

Comment:
This is great... but needs to answer the question... "Why?" Why should I care? When would I want to use XAML Navigation vs Navigation inside of my ViewModel.